plugin/lsp.vim

```diff
@@ -31,6 +41,7 @@ command! -nargs=0 -bar LspPeekImpl call lsp#gotoImplementation(v:true)
 command! -nargs=0 -bar LspShowSignature call lsp#showSignature()
 command! -nargs=0 -bar LspDiagShow call lsp#showDiagnostics()
 command! -nargs=0 -bar LspDiagCurrent call lsp#showCurrentDiag()
+command! -nargs=0 -bar LspDiagCurrentInStatusLine call lsp#showCurrentDiagInStatusLine()
```

Comment:
Can you update the help text with a description of this new command?

Reply:
I'll remove this one because LspDiagCurrent is enough for explicit calling of the command. This is only for automatic status list showing which won't interrupt a user by removing linebreaks.